src/db/models/request.js

```diff
@@ -1,19 +1,17 @@
 
 module.exports = (sequelize, DataTypes) => {
   const Request = sequelize.define('Request', {
-    userId: DataTypes.INTEGER,
+    managerId: DataTypes.INTEGER,
```

Comment:
This should be a foreign key referring to a user who is a manager.

Reply:
Solved!